qa/suites/rbd/migration-external/.qa

```diff
@@ -0,0 +1 @@
+../.qa/
```

Comment:
This and all other files that are intended to be symlinks are actually plain text files, so this won't schedule properly in teuthology.

Also, `conf` symlink is missing even in the form of a text file...  Need to redo the "set up the test skeleton by copying `migration` to `migration-external`" step from scratch.